cmd/yurthub/app/start.go

```diff
@@ -123,25 +144,8 @@ func Run(cfg *config.YurtHubConfiguration, stopCh <-chan struct{}) error {
 	}
 	trace++
 
-	klog.Infof("%d. create storage manager", trace)
-	storageManager, err := factory.CreateStorage()
-	if err != nil {
-		klog.Errorf("could not create storage manager, %v", err)
-		return err
-	}
-	storageWrapper := cachemanager.NewStorageWrapper(storageManager)
-	trace++
-
-	klog.Infof("%d. new serializer manager", trace)
-	serializerManager := serializer.NewSerializerManager()
-	trace++
-
-	klog.Infof("%d. new cache manager with storage wrapper and serializer manager", trace)
-	cacheMgr, err := cachemanager.NewCacheManager(storageWrapper, serializerManager)
-	if err != nil {
-		klog.Errorf("could not new cache manager, %v", err)
-		return err
-	}
+	klog.Infof("%d. run health checker for remote servers", trace)
+	healthChecker.Run()
```

Comment:
why not add `healthChecker.Run()` after `healthchecker.NewHealthChecker()`?

Reply:
fixed